modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlag.testcase

```diff
@@ -48,17 +48,17 @@ definition {
 		}
 
 		task ("When the administrator disables a feature flag") {
-			Check.checkToggleSwitch(
+			ClickNoError(
```

Comment:
Why is this needed?

Reply:
I think the logic was inverted there since `When the administrator disables a feature flag` we needed to `uncheck` instead of `check`